### PR TITLE
Add Skyrim VR

### PR DESCRIPTION
**Pull request by LunNova**
Had a go at adding a Skyrim VR game, with no changes from SE other than the paths and names

Seems to work although it's causing mod organiser's virtual file system to break if launched from it. Don't think that's related to these changes.

### CODE REVIEW

File: Mopy/bash/game/skyrimvr/__init__.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        u'skyrim.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'update.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'dawnguard.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'hearthfires.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'dragonborn.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa'],
+        u'skyrimvr.esm': [u'Skyrim - Patch.bsa', u'Skyrim - Interface.bsa', u'Skyrim_VR - Main.bsa'],
+    }
+
+    @classmethod
+    def init(cls):
```
Comment (nallar) [inline Mopy/bash/game/skyrimvr/__init__.py]: I don't get why this needed copy and pasted. without it it crashes. According to some google searches `@classmethod`s are inherited.

😖

File: scripts/mktaglist.py [hunk 2]
```diff
@@ -48,6 +48,7 @@ def DownloadMasterlist(repository, destinationPath):
     (u'Oblivion', 'Oblivion.esm', 'oblivion', loot_api.GameType.tes4),
     (u'Skyrim', 'Skyrim.esm', 'skyrim', loot_api.GameType.tes5),
     (u'Skyrim Special Edition', 'Skyrim.esm', 'skyrimse', loot_api.GameType.tes5se),
+    (u'Skyrim VR', 'Skyrim.esm', 'skyrimse', loot_api.GameType.tes5se),
```
Comment (nallar) [inline scripts/mktaglist.py]: There is no LOOT for skyrim VR yet so leaving this set to tes5se.

**Comment by Utumno** [comment]
Interesting - you should pack a standalone and post it to the forums for people to test it.

See:

https://github.com/wrye-bash/wrye-bash/blob/dev/Contributing.md#contributing-code

And for setting up the python environment to build Bash :+1: 
https://github.com/wrye-bash/wrye-bash/wiki/%5Bgithub%5D-Making-A-Release

**Comment by nallar** [comment]
setup.py is fussy about how it is started.

It was starting on python3 and crashing. Adding `#! python2` to `setup.template` made it start with python2 but then do nothing at all with no error.

For some reason changing the call in `package_for_release.py` to `subprocess.call(['py', '-2', setup, 'py2exe'], shell=True)` works. 😕

**Comment by nallar** [comment]
Asked for testing on the unofficial bethesda VR discord.

image

The earlier issue I mentioned with mod organiser 2 was due to [bugs in MO2](https://github.com/Modorganizer2/modorganizer/issues/264) which were fixed in the version released earlier today.

File: scripts/mktaglist.py (hunk 2 above)
Comment (Silarn) [reply scripts/mktaglist.py]: The latest loot API includes tes5vr now.

**Comment by Metallicow** [comment]
One of our guys has VR. Will try and test it out.

**Comment by nallar** [comment]
Once #408 works the LOOT API can be updated.

Can either wait for that to be fixed and merged, then update the API in this merge request, or I could add a TODO and it could be updated later.

I added the "Run mktaglist" commit as committing those changes was recommended as the steps for a build. As it was only for a test build and this is a merge request, not sure I actually should have committed that.

**Comment by Infernio** [comment]
Closed in favor of #454.

**Pull request closed without merging**